Doc/library/sqlite3.rst

```diff
@@ -1316,31 +1317,13 @@ Connection objects
 
    .. attribute:: row_factory
 
-      A callable that accepts two arguments,
-      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
-      and returns a custom object representing an SQLite row.
-
-      Example:
-
-      .. doctest::
-
-         >>> def dict_factory(cursor, row):
-         ...     col_names = [col[0] for col in cursor.description]
-         ...     return {key: value for key, value in zip(col_names, row)}
-         >>> con = sqlite3.connect(":memory:")
-         >>> con.row_factory = dict_factory
-         >>> for row in con.execute("SELECT 1 AS a, 2 AS b"):
-         ...     print(row)
-         {'a': 1, 'b': 2}
-
-      If returning a tuple doesn't suffice and you want name-based access to
-      columns, you should consider setting :attr:`row_factory` to the
-      highly optimized :class:`sqlite3.Row` type. :class:`Row` provides both
-      index-based and case-insensitive name-based access to columns with almost no
-      memory overhead. It will probably be better than your own custom
-      dictionary-based approach or even a db_row based solution.
+      Define the default :attr:`Cursor.row_factory`
+      for cursors created off of this connection.
```

Comment:
```suggestion
      The default :attr:`~Cursor.row_factory`
      for :class:`Cursor` objects created from this connection.
```

* Using a verb like "Define" makes it sound like this is a callable rather than an attribute, so I suggest eliding it.
* I'd suggest omitting the class here and adding explicit markup to the `Cursor` mention following it, as it reads better to me and avoids redundancy
* IMO, "from" rather than "off of" is a little clearer at representing the relationship here

Reply:
Totally agree on every point. That reads much better; thanks.